packages/playout-gateway/src/atemUploader.ts

```diff
@@ -39,29 +40,44 @@ export class AtemUploadScript {
 		})
 	}
 
-	public loadFile(): Promise<Buffer> {
+	public loadFile(fileUrl: string): Promise<Buffer> {
 		return new Promise<Buffer>((resolve, reject) => {
-			fs.readFile(this.fileUrl, (e, data) => {
+			fs.readFile(fileUrl, (e, data) => {
 				consoleLog('got file')
 				if (e) reject(e)
 				else resolve(data)
 			})
 		})
 	}
+	public async loadFiles(folder: string): Promise<Buffer[]> {
+		const files = await fs.promises.readdir(folder)
+		consoleLog(files)
+		const loadBuffers = files.map((file) => fs.promises.readFile(path.join(folder, file)))
+		const buffers = await Promise.all(loadBuffers)
```

Comment:
This await is unnecessary, given that the function returns a `Promise<Buffer[]>` anyway